app/src/main/java/org/simple/clinic/editpatient/EditablePatientEntry.kt

```diff
@@ -142,7 +142,10 @@ data class EditablePatientEntry @Deprecated("Use the `from` factory function ins
       copy(alternativeId = alternativeId)
 
   fun addBpPassports(bpPassports: List<Identifier>): EditablePatientEntry {
-    return copy(bpPassports = this.bpPassports.orEmpty() + bpPassports)
+    return if (this.bpPassports.orEmpty().contains(bpPassports.last()))
+      copy(bpPassports = this.bpPassports.orEmpty())
+    else
+      copy(bpPassports = this.bpPassports.orEmpty() + bpPassports)
```

Comment:
@Honey14 Since this is business logic, can we move this to the `Update` function, please?